config/config.js

```diff
@@ -0,0 +1,34 @@
+export default {
+  development: {
```

Comment:
Shall we extract this out into something like:  


```
const baseConfig = {
  username: process.env.DB_USERNAME,
  password: process.env.DB_PASSWORD,
  database: process.env.DB_NAME,
  host: process.env.DB_HOSTNAME,
  port: process.env.DB_PORT,
  dialect: 'postgres',
}
```

Since all the envs are using the same params